company-profiles/massivepixelcreation.md

```diff
@@ -0,0 +1,18 @@
+# Massive Pixel Creation
```

Comment:
This company title doesn't match how it is in the main readme file. Either the name is three separate words or it's CapitalCase.

If the name is separate then this file should be `massive-pixel-creation.md`